### PR TITLE
Use PEP8 naming

### PR DESCRIPTION
**Pull request by EarlMilktea**
**Description of the change:**

Rename everything violating PEP8 naming convention.

### CODE REVIEW

File: graphix/visualization.py [hunk 1]
```diff
@@ -62,11 +62,11 @@ def __init__(
         local_clifford : dict
             dict specifying the local clifford for each node.
         """
-        self.G = G
+        self.g = g
```
Comment (thierry-martinez) [inline graphix/visualization.py:65]: A longer variable or field name, such as `graph`, would be better than `g`.

**Review by thierry-martinez — CHANGES_REQUESTED** [review]
Looks good to me. I just made two minor remarks.

File: graphix/linalg.py [hunk 2]
```diff
@@ -232,44 +232,44 @@ def forward_eliminate(self, b=None, copy=False):
...
+            mat_a = self
         if b is None:
-            b = np.zeros((A.data.shape[0], 1), dtype=int)
+            b = np.zeros((mat_a.data.shape[0], 1), dtype=int)
         b = MatGF2(b)
         # Remember the row and column order
-        row_permutation = [i for i in range(A.data.shape[0])]
-        col_permutation = [i for i in range(A.data.shape[1])]
+        row_permutation = [i for i in range(mat_a.data.shape[0])]
+        col_permutation = [i for i in range(mat_a.data.shape[1])]
```
[the remark this answers is not in the document]
Comment (EarlMilktea) [reply graphix/linalg.py:243]: This kind of simplifications can be performed uniformly with additional Ruff rules.
Let me submit another PR for that.

File: graphix/visualization.py (hunk 1 above)
Comment (EarlMilktea) [reply graphix/visualization.py]: Fixed in 4780cd3b92f8c4f1569707257511c42b12984b28 .

Comment (EarlMilktea) [reply graphix/visualization.py]: Fixed in 0ee3b217e1dd5db66213ddfb6b17da5a7f087df8 .

**Review by thierry-martinez — APPROVED** [review]

**Review by masa10-f — APPROVED** [review]
Looks good to me!

**Comment by shinich1** [comment]
@EarlMilktea thanks! please make sure docs build without error, and add a line or two to CHENGELOG.md.

**Comment by EarlMilktea** [comment]
@shinich1 Done!

**Comment by shinich1** [comment]
@EarlMilktea please add in [unreleased] part.

**Review by shinich1 — APPROVED** [review]

**Pull request merged**